Directions.md

```diff
@@ -86,3 +86,37 @@ Bon apetit!
 16) Add 2-3 table spoons of pomegranate sauce on top
 
 [Making of Ezme Salad](https://www.youtube.com/watch?v=03kbOZxa3oU)
+
+### Barbecue Veggies (Veggies on Skewers)
+
+1) Cut the tomatoes and onions into half and keep their skin on
```

Comment:
Should we soak the skewers or veggies in a marinade or oil to prevent burning?  If the skewers are metal, this is not required for them.